curriculum/challenges/english/01-responsive-web-design/accessibility-quiz/part-045.md

```diff
@@ -0,0 +1,258 @@
+---
+id: 614796cb8086be482d60e0ac
+title: Part 45
+challengeType: 0
+dashedName: part-45
+---
+
+# --description--
+
+On the topic of visual accessibility, contrast between elements is a key factor. For example, the contrast between the text and the background of a heading should be at least 4.5:1.
+
+Change the font color of all the anchor elements within list elements to something with a contrast ratio of at least 7:1.
+
+# --hints--
+
+You should use the `li > a` selector.
+
+```js
+assert.exists(new __helpers.CSSHelp(document).getStyle('li > a'));
+```
+
+You should give the `a` element a `color` property.
+
+```js
+assert.notEmpty(new __helpers.CSSHelp(document).getStyle('li > a')?.color);
+```
+
+You should give the `color` property a contrast with the background of at least 7:1. _Hint: I would use `#dfdfe2`_
```

Comment:
Using `white` is cheating 😜 

I will try to find a way to convert all valid inputs into RGB.

Reply:
Not cheating, but definitely lazy lol.

I think it would be fair to exclude them, if we specify in the text somewhere that "you should give the `color` property a hex value with a contrast..."